File: practice_java_program/notpad/RevString.java
public class RevString{
	public static void main(String[] args){
		String str = "United state of america";
		char[] arr = str.toCharArray();
		int start = 0;
		int end = str.length()-1;
		while(start<end){
			char temp=arr[start];
			arr[start]=arr[end];
			arr[end]=temp;
			
			start++;
			end--;
		}
		System.out.println(new String(arr));
	}
}

File: python_oriented_interview/notpad/RevString.py
str_val = "United state of america"
arr = list(str_val)
start = 0
end = len(str_val) - 1
while start < end:
    temp = arr[start]
    arr[start] = arr[end]
    arr[end] = temp
    start += 1
    end -= 1

print("".join(arr))
